tsconfig.json

```diff
@@ -18,6 +18,8 @@
     "importHelpers": true,
     "target": "es2020",
     "module": "es2020",
+    "resolveJsonModule": true,
+    "esModuleInterop": true,
```

Comment:
Comment: This was necessary for the json to work as its own file. I am unsure if that was the right move. Initially I had the json data in the component and then later moved it to come in as a service.

Reply:
not sure on this one :)